examples/DynaFlow/IEEE14/IEEE14_DisconnectLine/IEEE14.jobs

```diff
@@ -13,22 +13,18 @@
 -->
 <dyn:jobs xmlns:dyn="http://www.rte-france.com/dynawo">
   <dyn:job name="IEEE14 - Disconnect Line">
-    <dyn:solver lib="dynawo_SolverSIM" parFile="solvers.par" parId="3"/>
+    <dyn:solver lib="dynawo_SolverSIM" parFile="IEEE14.par" parId="SimplifiedSolver"/>
     <dyn:modeler compileDir="outputs/compilation">
-      <dyn:network iidmFile="IEEE14.iidm" parFile="IEEE14.par" parId="8"/>
+      <dyn:network iidmFile="IEEE14.iidm" parFile="IEEE14.par" parId="Network"/>
       <dyn:dynModels dydFile="IEEE14.dyd"/>
       <dyn:precompiledModels useStandardModels="true"/>
       <dyn:modelicaModels useStandardModels="true"/>
     </dyn:modeler>
-    <dyn:simulation startTime="0" stopTime="100"/>
+    <dyn:simulation startTime="0" stopTime="1000"/>
     <dyn:outputs directory="outputs">
-      <dyn:dumpInitValues local="true" global="true"/>
-      <dyn:curves inputFile="IEEE14.crv" exportMode="CSV"/>
```

Comment:
Note: you removed the curves here, which leads to an error in the nrt because you added a curves file in the references

Reply:
I reintroduced them in order to have a reference to compare to.